packages/kbn-optimizer/src/integration_tests/bundle_cache.test.ts

```diff
@@ -83,6 +83,11 @@ it('emits "bundle cached" event when everything is updated', async () => {
 
   expect(cacheEvents).toMatchInlineSnapshot(`
     Array [
+      Object {
+        "bundle": <Bundle>,
+        "reason": "missing optimizer cache key",
+        "type": "bundle not cached",
+      },
       Object {
         "bundle": <Bundle>,
         "type": "bundle cached",
```

Comment:
@joshdover Can we add the `src/core/public/entry_point.ts` fixture to a new mock repo or something to avoid all the changes to the other tests? All of the `"bundle not cached"` notifications are concerning.